tests/misc_env/push_cosbench_workload.py

```diff
@@ -6,21 +6,25 @@
     - test:
         abort-on-fail: true
         clusters:
-          ceph:
+          ceph-pri:
```

Comment:
@hmaheswa, can we create another .py for multisite env and retain the existing for single site.

Reply:
@mkasturi18 , this py iteslf works for both single multisite env
pass logs for single site cluster
http://magna002.ceph.redhat.com/ceph-qe-logs/HemanthSai/record_sync_baremetal_automation/cephci-run-XPPO4H-singlesite/

conf:
https://github.com/red-hat-storage/cephci/blob/master/conf/quincy/rgw/tier-0_rgw.yaml

test:
`  - test:
      abort-on-fail: true
      config:
        controllers:
          - node5
        drivers: # if drivers are not specified will use one of the rgw node
          - node5
          - node6
        fill_percent: 0.03
      desc: prepare and push cosbench fill workload
      module: push_cosbench_workload.py
      name: push cosbench fill workload`